services/tasks/local_unix.go

```diff
@@ -0,0 +1,56 @@
+// +build !windows_v2
```

Comment:
is `windows_v2` being used here for shorthand that it imples `!windows`; assuming that a `Makefile` section at some point marries the `BUILDTAG` addition of `windows_v2` to only be turned on when `GOOS=windows`?

I only ask because in your windows source file you were explicit to join `windows` and `windows_v2`. Not a big deal, but curious as maybe they should both use the same strategy, even though I think build-time constraints will most likely make it immaterial.

Reply:
@estesp - Actually, in this case `local_unix.go` is confusing. This really is `local_all.go` where `all` is everything that exists today `!windows_v2`. I named it this way though because when Windows v1 is removed this would be only for Unix. You can see in [Makefile.windows#L30](https://github.com/containerd/containerd/blob/master/Makefile.windows#L30) the following:
```make
ifeq (${BUILD_WINDOWS_V2},1)
	BUILDTAGS += windows_v2
endif
```
So `windows_v2` is really `GOOS=windows` and an additional tag `-tags windows_v2`. It is used to flip/flop between the Windows runtime v1 service and the new runtime v2 `TasksService` while the implementation is completed. Then the Windows v1 runtime will be removed.